pkg/tsdbctl/tsdbctl_test.go

```diff
@@ -52,11 +52,11 @@ func (suite *testTsdbctlSuite) TestPopulateConfigWithTenant() {
 
 	expectedRc := RootCommandeer{
 		v3iocfg:  cfg,
-		v3ioPath: "localhost:80123/123",
+		v3ioUrl:  "localhost:80123",
```

Comment:
how is the test passes if you deleted the `container` from the expected and not from the "input"

Reply:
Because it goes in the `container` field, not `v3ioUrl`.